client/src/pages/SettingsPage/components/AdminSettingsPage/AdminSettings.jsx

```diff
@@ -0,0 +1,136 @@
+import { React, useState }  from 'react';
+import routes from '../../../../configs/routes';
+import { userSelector } from '../../../../redux';
+import { Link } from 'react-router-dom';
+import resset_password_icon from '../../../../assets/images/SettingsPageImages/resset_password_image.svg';
+import username_icon from '../../../../assets//images/SettingsPageImages/username-icon.svg';
+import changes_was_saved_icon from '../../../../assets/images/SettingsPageImages/changes_was_saved.svg';
+import cancel from '../../../../assets/images/SettingsPageImages/cancel.svg';
+import style from './AdminSettings.module.scss';
+import SaveButton from '../../../../shared/Buttons/Save';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchUpdateUser } from '../../../../redux/requests/userRequest';
+
+const AdminSettings = () => {
+
+  const dispatch = useDispatch();
+
+  const user = useSelector(userSelector);
+
+  const [isShowChanges, setShowChanges] = useState(false);
+  
+  const [firstName, setFirstname] = useState('');
```

Comment:
in firstName and lastName you can get data from redux, like:
```suggestion
  const [firstName, setFirstname] = useState(user.firstName);
```